src/components/implementation/tests/micro_booter/test_ipi_n_n.c

```diff
@@ -0,0 +1,181 @@
+#include <stdint.h>
+
+#include "micro_booter.h"
+
+extern int _expect_llu(int predicate, char *str, long long unsigned a, long long unsigned b, char *errcmp, char *testname, char *file, int line);
+extern int _expect_ll(int predicate, char *str, long long a, long long b, char *errcmp, char *testname, char *file, int line);
+
```

Comment:
I think the idea here is to measure 1 to 1 IPC latency between each combination of cores and display it like a matrix of numbers? 
perhaps both inter-core and intra-core.

Reply:
We don't seem to measure anything here in this test.